qlib/contrib/model/pytorch_tabnet.py

```diff
@@ -410,9 +406,9 @@ def forward(self, x, priors):
 
 
 class DecoderStep(nn.Module):
-    def __init__(self, inp_dim, out_dim, shared, n_ind, vbs, device):
-        super().__init__()
-        self.fea_tran = FeatureTransformer(inp_dim, out_dim, shared, n_ind, vbs, device)
+    def __init__(self, inp_dim, out_dim, shared, n_ind, vbs):
+        super(DecoderStep, self).__init__()
```

Comment:
This is not necessary in Python3

Reply:
Should we follow PyTorch style for these PyTorch models? From FB's official implementation of different PyTorch models (https://github.com/pytorch/vision/blob/master/torchvision/models/googlenet.py#L73), they all use such style.

Reply:
It is not necessary in fact. The PyTorch style is only needed for inheritance of `nn.Module` class, while in `Qlib` we inherit from our customized `Model` class.

Reply:
This class is the subclass of `nn.Module` not `qlib.Model`~~

Reply:
Same as above.